test/powershell/Modules/Microsoft.PowerShell.Utility/Set-Date.Tests.ps1

```diff
@@ -10,6 +10,12 @@ Describe "Set-Date" -Tag "CI" {
         { get-date | set-date } | Should not throw
     }
 
+    It "Set-Date should be able to set the date with -Date parameter" -Skip:(! $IsElevated) {
+        $target = Get-Date
+        $expected = $target
+        Set-Date -Date $target | Should be $expected
```

Comment:
Please replace `Should be` with `Should Be` - it is our pattern.

Reply:
In the [document](https://github.com/PowerShell/PowerShell/blob/v6.0.0/docs/testing-guidelines/WritingPesterTests.md), `Be` and `be` are mixed.
And the other tests in `Set-Date.Tests.ps1` is described by `be` .
Should I rewrite all tests?

Reply:
I'd happy if you'd fix all bad formatting in the file with separate commit.
get-date | set-date -> Get-Date | Set-Date 
should be -> Should Be

Reply:
I fixed the code.